covalent/_shared_files/schemas/electron.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+"""FastAPI models for /api/v1/resultv2 endpoints"""
+
+from datetime import datetime
+from typing import Dict, Optional
+
+from pydantic import BaseModel, validator
+
+from .asset import AssetSchema
+from .common import StatusEnum
+
+ELECTRON_METADATA_KEYS = {
+    "task_group_id",
+    "name",
+    "start_time",
+    "end_time",
+    "status",
+    # electron metadata
```

Comment:
what does this mean if the whole dict is metadata?

Reply:
ELECTRON_METADATA_KEYS and ELECTRON_ASSET_KEYS segregate the attributes of each electron into metadata or assets. Metadata attributes map to columns in the Electrons table and will end up in the manifest in their entirety. Asset attributes are serialized to a staging directory for separate uploads.

Reply:
The sets ELECTRON_METADATA_KEYS and ELECTRON_ASSET_KEYS segregate the attributes of each electron into metadata or assets. Metadata attributes map to columns in the Electrons table and will end up in the manifest in their entirety. Asset attributes are serialized to a staging directory for separate uploads.

Reply:
The sets ELECTRON_METADATA_KEYS and ELECTRON_ASSET_KEYS partition the attributes of each electron into metadata or assets. Metadata attributes map to columns in the Electrons table and will end up in the manifest in their entirety. Asset attributes are serialized to a staging directory for separate uploads. The `electron_metadata` comment refers to the `metadata` parameter of electron decorators -- executor, deps.